### DISCUSSION TITLE
Is it possible to include font size property to the component

### DISCUSSION THREAD

**Discussion by Padhysai** [microsoft/powercat-creator-kit#77] [Q&A] [answered]
Hello All,
Is there any flexibility to change the font type and size of the text. When using with high-resolution canvas apps(1920x1080), text is barely visible and difficult to read.

**Comment by denise-msft** [comment, marked as answer]
Hi @Padhysai 

Today we do not currently support changing font size to stay consistent with the Fluent UI design principals that are inherent in the control.

If you zoom into the app by using the browser's zoom capabilities, and if you are using [responsive app design](https://docs.microsoft.com/en-us/power-apps/maker/canvas-apps/create-responsive-layout) components and implementation, you can achieve a larger font size. This design will make sure that your application is also accessible to people who choose to render the text size larger anyways.

Can you please try that suggestion and let us know if it's not a solution for you?

Because you are not the first person to ask for this capability, if there is a particular set of components that you would like the ability to configure font size, please share those with us so we can consider them for one of our next sprints.

**Reply by EnriqueSanchezM** [reply]
I imgine that this solution is not supported, but I have found the way to increase the font-size in Command Bar. This is my post: https://esanchezm.home.blog/2022/09/22/creator-kit-modificar-la-fuente-del-command-bar/

**Reply by denise-msft** [reply]
Correct, this would not be supported and not recommended for production applications, because this is not behavior we configured and support.

Did you try checking how big the components look when you play the app? Unlike the studio components, the creator kit components are rendered as the correct size in play mode (at least in the studio), even though they appear to be much smaller when you adjust the display resolution in the studio.

Can you please check if the commandBar is still too small when the app is played, and validate that the "Scale to fit" option is disabled?
